src/main/resources/schemas/schema-form.json

```diff
@@ -58,6 +58,18 @@
                 "expression-language": true
             }
         },
+        "connectTimeout": {
+            "title": "JWKS URL connect timeout",
+            "description": "Only applies when the resolver is JWKS_URL",
+            "type": "integer",
+            "default": 2000
+        },
+        "requestTimeout": {
+            "title": "JWKS URL request timeout",
+            "description": "Only applies when the resolver is JWKS_URL",
```

Comment:
I's now possible do configure a gioConfig to hide this option according another props value.
Just FYI ;)